samples/SDCard/app/application.cpp

```diff
@@ -106,7 +106,7 @@ void stat_file(char* fname)
 }
 
 FRESULT ls(const char* path /* Start node to be scanned (also used as work area) */
-)
+		   )
```

Comment:
Don't modify this file in the current PR.

Reply:
Fixed